xbmc/rendering/MatrixGL.neon.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+
+#if defined(HAS_NEON) && !defined(__LP64__)
```

Comment:
remove

Reply:
I've tried to remove it, but it failed, this code is triggered on IOS-ARM64 despite the cmake condition. cmake seems weak to detect host versus target capabilities.
The other yuv2rgb.neon.S seems to work fine on aarch64.

Reply:
I will try to rework the cmake condition anyway.